applications/RomApplication/custom_strategies/rom_builder_and_solver.h

```diff
@@ -676,6 +410,347 @@ class ROMBuilderAndSolver : public BuilderAndSolver<TSparseSpace, TDenseSpace, T
         }
     }
 
+
+    void InitializeHROMWeights(ModelPart& rModelPart)
+    {
+        KRATOS_TRY
+
+        using ElementQueue = moodycamel::ConcurrentQueue<Element::Pointer>;
+        using ConditionQueue = moodycamel::ConcurrentQueue<Condition::Pointer>;
+
+        // Inspecting elements
+        ElementQueue element_queue;
+        DofsVectorType tmp_dof_list; // Preallocation
+        block_for_each(rModelPart.Elements().GetContainer(), tmp_dof_list,
+            [&](Element::Pointer p_element, DofsVectorType& dof_list)

```

Comment:
```suggestion
        block_for_each(rModelPart.Elements().GetContainer(), [&](Element::Pointer p_element)
```
I think the DOF list is a leftover in here.

Reply:
Yes it is. Removed.